webpages/CreateAccount.php

```diff
@@ -0,0 +1,3 @@
+<?php
+header('Location: ./ForgotPasswordLink.php?' . $_SERVER['QUERY_STRING']);
```

Comment:
I'm being a little bit lazy here. I wanted to have a URL in the email that looked like it was about creating an account, but the code is substantially similar to resetting your password. If we wanted to, we could pull all of the password reset link handling into some reusable code that is used by both `PasswordResetLink.php` and `CreateAccount.php`.

Reply:
Ah. I confess that I was a bit confused by this file.